src/pages/App.tsx

```diff
@@ -80,16 +82,60 @@ function getCurrentPageFromLocation(locationPathname: string): PageName | undefi
   }
 }
 
+// Get browser being used, code comes from: https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator.
```

Comment:
can this live in `utils/browser.ts` or something similar?

Reply:
doesn't amplitude handle this for us? i think they detect browser

Reply:
amplitude doesn't, indicated in their documentation (https://help.amplitude.com/hc/en-us/articles/215562387-User-property-definitions) and we also emailed with them early on to verify. moved to utils/browser.ts